src/query/api/v1/handler/headers.go

```diff
@@ -56,6 +56,12 @@ const (
 	// metrics type.
 	MetricsStoragePolicyHeader = "M3-Storage-Policy"
 
+	// FetchRestrictLabels restricts all fetches to match certain labels.
+	FetchRestrictLabels = "M3-Fetch-Restrict-Labels"
+
+	// FetchStripLabels strips certain labels from the result.
+	FetchStripLabels = "M3-Fetch-Strip-Labels"
```

Comment:
Think we were moving towards using just one header, called `M3-Query-Options-JSON`?

Perhaps:
```go
QueryOptionsJSONHeader = "M3-Query-Options-JSON"
```

Reply:
Oh yeah oops forgot to prune this one

Reply:
Moving to `M3-Restrict-By-Tags-JSON`